packages/webview_flutter/webview_flutter_web/lib/src/web_webview_controller.dart

```diff
@@ -18,19 +18,32 @@ import 'http_request_factory.dart';
 @immutable
 class WebWebViewControllerCreationParams
     extends PlatformWebViewControllerCreationParams {
-  /// Creates a new [AndroidWebViewControllerCreationParams] instance.
+  /// Creates a new [WebWebViewControllerCreationParams] instance.
+  /// [iFrameCredentialless] can be used to set the credentialless attribute on the <iframe>.
   WebWebViewControllerCreationParams({
     @visibleForTesting this.httpRequestFactory = const HttpRequestFactory(),
-  }) : super();
+    bool iFrameCredentialless = false,
+  })  : iFrame = web.HTMLIFrameElement()
+          ..id = 'webView${_nextIFrameId++}'
+          ..style.width = '100%'
+          ..style.height = '100%'
+          ..style.border = 'none'
+          ..setAttribute(
+              'credentialless', iFrameCredentialless ? 'true' : 'false'),
```

Comment:
Setting `credentialless` to any value results it in being `true`

* https://jsfiddle.net/ditman/obufjvtk/

Set the attribute only `iFrameCredentialless` is `true`.